test/test_types.py

```diff
@@ -32,6 +33,7 @@ def test_basic_target():
     assert t.alphanumeric_color == Color.NONE
     assert t.image is None
     assert t.confidence == 0.0
+    assert str(t) == "Target(x=2, y=4, width=5, height=8, orientation=0.0, confidence=0.0, shape=Shape.NAS, color=Color.NONE, alphanumeric=, alphanumeric_color=Color.NONE)"
```

Comment:
The alpha should probably be put in single quotes so this makes it a valid expression